### PR TITLE
PRIME-2368 Self Declaration Version not set for paper enrolment

### PR DESCRIPTION
**Pull request by bergomi02**
This item to address that SelfDeclaration.SelfDeclarationVersionId is not set for paper enrolement.

### CODE REVIEW

File: prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs [hunk 1]
```diff
@@ -220,6 +220,24 @@ public async Task UpdateSelfDeclarationsAsync(int enrolleeId, IEnumerable<PaperE
...
+            {
+                var versions = await _context.Set<SelfDeclarationType>()
+                    .AsNoTracking()
+                    .Select(t => _context.Set<SelfDeclarationVersion>()
+                        .Where(av => av.EffectiveDate <= enrollee.SelfDeclarationCompletedDate)
+                        .Where(av => av.SelfDeclarationTypeCode == t.Code)
+                        .OrderByDescending(av => av.EffectiveDate)
+                        .First())
+                    .OrderBy(av => av.SelfDeclarationType.SortingNumber)
+                    .ToListAsync();
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs:233]: This code seems to be in three other locations.  Can `LookupService,GetSelfDeclarationVersion` be used instead?

Comment (bergomi02) [reply prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs]: As suggested, the code is updated to use LookupService to pull the versions in different locations.

**Comment by neophyte57** [comment]
Do not merge until confirm Paper Enrolment functionality will be used (due to testing effort)

**Comment by bergomi02** [comment]
as paper enrolment is not in used, closing this PR.

**Pull request closed without merging**